### ISSUE TITLE
hepaticSpatialProteogenomics - Guilliams

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/hca-ebi-wrangler-central#666]
### Project short name:
hepaticSpatialProteogenomics

### Primary Wrangler:
Irene Perez-Diez

### Secondary Wrangler:
Ida

### Associated files

* Google Drive: https://drive.google.com/drive/folders/1g0h738T-dY9VQ-GQrx9OWPSJdzO5jkHv
* Ingest: https://contribute.data.humancellatlas.org/projects/detail?uuid=425c2759-db66-4c93-a358-a562c069b1f1

### Published study links

* Paper:https://www.sciencedirect.com/science/article/pii/S0092867421014811?via%3Dihub#sec5

* Accessioned data: 
  - [GSE192740](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192740) 
  - [GSE192742](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192742)
  - [GSE192741](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192741)

### Key Events

- [x] Convert published metadata to HCA spreadsheet
- [x] Manually curate dataset to meet HCA metadata standard
- [x] Collect any matrix and cell-type annotation files
- [x] Upload sheet to validate metadata
- [x] Check linking using ingest graph validator
- [x] Transfer raw files to ingest to validate data files
- [x] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [x] Submit dataset to Production 
- [x] Complete the Export SOP
- [ ] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Comment by ESapenaVentura** [comment]
Please note this dataset contains other organisms' data (e.g. Zebrafish)

Please don't include these when wrangling

**Comment by ipediez** [comment]
Needs term request:

- [ ] Liver abscess, as child of [abscess [MONDO:0005227]](https://ontology.archive.data.humancellatlas.org/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0005227)

**Comment by ipediez** [comment]
Patients have several diseases, for the specimen level I have only kept those diseases that affect directly the liver

**Comment by gabsie** [comment]
blocked because of the schema update

**Comment by idazucchi** [comment]
Hi!
I'm done with the secondary review. You were right about 666 being the appropriate number for this dataset: congratulations on wrangling this!

I've made some notes, most of the dataset is all right, there are a few instances where you swapped the protocols by mistake and some other notes are about information that could be added either to the donors or to the protocols.
Under Specimen from donor there are a lot of notes because I would model the specimens differently and I tried to write it down in a way that would be easy to read.
I'm sorry it's so long but I promise it's not bad. 
If there is anything that you want to discuss let me know!

### Project info
- some grants are missing

### Donor
- The weight and height are missing for all humans but they all have the BMI: is this to reduce the identifying information for the donors?
- can we record in the disease field the steatosis>10% (I'm not sure we have an ontology term), as well as the fibrosis status (we have an ontology term for it)? 
**Mice**
- the sex is missing for all mice, you can put `unknown` 
- the `developmental stage` for all mice is adult
- mice age is either 36+5 or 24+5 weeks
- all the mice dead are dead at collection
- for visium mice: `cause of death` euthanised with co2
- western diet mice:
   - sex is `male` according to the paper
   - I would change the disease to `non-alcoholic fatty liver disease` MONDO:0013209 and `non-alcoholic steatohepatitis` MONDO:0007027 to reflect the effects of the diet
- pbs mice:
   - they are 29 weeks old according to the paper
   - you could add to the description for the time course or maybe ask the authors? It’s not really clear what they did, in the paper they mention a time frame of 2h and in geo is 24h
> Mice were treated with 3.5 mg/kg PBS and 2 h later, livers were harvested without the capsule 

### Collection protocol
- you could use the term biopsy for `liverBiopsy`

### Specimen from donor organism
**mice**
I would model the specimens differently, most of the notes are about that
- for visium mice:
   - `healthyMouseLiver_5` should have 4 specimens, then corresponding to 1 imaged specimen each
- healthy mice 10x: should be a total of 12 specimens
   - 2 more specimens for `healthyMouseLiver_1`
   - 1 more specimen for `healthyMouseLiver_2`
   - 1 more specimen for `healthyMouseLiver_3`
   - 3 more specimens for `healthyMouseLiver_4`
   - 2 more specimens for `perfusedHealthyMouseLiver_1`
   - 2 more specimens  for `perfusedHealthyMouseLiver_2`
   - 1 more specimen for `perfusedHealthyMouseLiver_3`
   - `healthyMouse_6` and `healthyMouse_8` are donors for ex vivo digestion, so they shouldn’t be perfused mice, this carries over to the cell suspension. Their collection protocol should be `liverExtraction`
- standard diet mice:
   - `sdMouseLiver_2` should have `liverPerfusionExtraction` for collection protocol
   - I would add 2 more specimens for `sdMouseLiver_1`
- western diet mice:
   - I would change the disease to `non-alcoholic fatty liver disease` MONDO:0013209 and `non-alcoholic steatohepatitis` MONDO:0007027
- pbs mice:
   - they extracted just the liver, without the capsule. I don't think we have an ontology term for that, maybe you could add it to the description
   - `pbsMouseLiver_2` should have `liverPerfusionExtraction` for collection protocol

 **humans**
 - can we record in the disease field the steatosis>10% (I'm not sure we have an ontology term), as well as the fibrosis status (we have an ontology term for it)? 
- how many specimen come from H02? it's not clear from the information in GEO
- I think `humanLiver_H14` has known disease `Hypercholesterolemia` not `hyperlipidemia`
- should `humanLiver_H18, humanLiver_H07, humanLiver_H21` have ‘adenoma’ in the disease field? The paper says explicitly that liver biopsies were isolated from healthy adjacent tissue removed during liver resection due to colorectal cancer metastasis, but there is no information directly for liver adenoma
- for donors `H37` and `H38` do we assume that one sample was used both for visium and 10x? I would model it with separate specimens

### Dissociation protocol
- you could add the publication doi for nuclei dissociation
- I would add to description of `enzymaticDissociationInVivo` the perfusion step, because it’s already an enzymatic digestion
- you could add a mechanical dissociation protocol to be applied to both in and ex vivo digestion, to indicate that the liver was cut up 

### Enrichment protocol
- I would delete `FACS_nuclei` and fill in the cell viability instead because it’s not selecting a certain type of cell?  Alternatively add the positive sign to DAPI

### Cell suspension
- INSDC sample accessions are missing
- for cells suspensions that were FACS-enriched the cell viability was assessed with Fixable viability dies
**mice**
- healthy mice 10x: 
   - the specimen for `GSM5764250` should be `healthyMouseLiverCapsule_1`
   - the digestion protocol should be `enzymaticDissociationExVivo` for `GSM5764270, GSM5764271, GSM5764274, GSM5764275, GSM5764278, GSM5764279, GSM5764285`
- standard diet mice:
   - the digestion protocol should be `enzymaticDissociationInVivo` for `GSM5764260, GSM5764261`
   - the size enrichment is missing for `GSM5764254, GSM5764255, GSM5764256, GSM5764257, GSM5764260, GSM5764261, GSM5764262, GSM5764263, GSM5764288, GSM5764290, GSM5764291, GSM5764294, GSM5764296`
- western diet mice:
   - the size enrichment is missing for `GSM5764289, GSM5764292, GSM5764293, GSM5764295, GSM5764297`
- pbs mice:
   - the digestion protocol should be `enzymaticDissociationInVivo` for `GSM5764264, GSM5764265, GSM5764266, GSM5764267`
   - the size enrichment is missing for `GSM5764258, GSM5764259, GSM5764264, GSM5764265, GSM5764266, GSM5764267`
**humans**
- `GSM5764394, GSM5764395, GSM5764396, GSM5764397, GSM5764398, GSM5764399, GSM5764400, GSM5764401` should have `mechanicalDissociationNuclei` for dissociation protocol
- the size enrichment is missing for `GSM5764338, GSM5764341, GSM5764344, GSM5764346, GSM5764349, GSM5764351, GSM5764354, GSM5764356, GSM5764358, GSM5764361, GSM5764363, GSM5764366, GSM5764368, GSM5764370, GSM5764373, GSM5764375, GSM5764377, GSM5764380, GSM5764383, GSM5764384, GSM5764385, GSM5764386, GSM5764387, GSM5764388, GSM5764389, GSM5764390, GSM5764391, GSM5764392, GSM5764393`

### Imaging preparation protocol
- consider splitting the imaging protocols to indicate clearly which slides were marked with antibodies (just `GSM5764423`)

### Imaged specimen
- `GSM5764414, GSM5764415, GSM5764416, GSM5764417` should have different specimens as input, all from the same donor
- you could add the INSDC sample accessions

### Imaging protocol
- the imaging indications hopefully apply to both Abs+/- slides

### Image file
- all right!! 
I'm sorry this is so long :(

### Library preparation
- visium
  - consider adding an additional visium protocol for the slides that were not Abs-marked
  - you can fill in the kit manufacturer and leave just the kit name in the retail name section
- cite-seq: you can add the description of the blocking abs
> When CITE-seq was to be performed, cells were then stained with 2.4G2 antibody to block Fc receptors and CITE-seq antibodies for 20mins at 4C, before being washed in excess PBS with 2% FCS and 2mM EDTA. Antibody details are included in the key resources table

### Sequence file
- these cell suspensions `GSM5764310, GSM5764312, GSM5764314, GSM5764315, GSM5764316, GSM5764323, GSM5764327` should have `lp_nucleiv3` as library preparation

### Analysis protocol
- the ontology term is missing
- computational methods are missing
- derivation process is missing
- the description doesn’t include filtering but the files are all `filtered_features_bc`, should this be a processed matrix protocol?

### Analysis file
- the  imaged specimen input for `GSM5764423_counts_ADT_BH001.csv.gz` should be `GSM5764423`
- for the analysis files from Cite-seq you list both cite-seq and 10x library prep: isn’t cite-seq enough?
- the cell count matrices are missing for all the human 10x CS, did you exclude them for a particular reason?

**Comment by ofanobilbao** [comment]
@ipediez and @idazucchi to have a meeting to go over some of the modelling. Irene has applied most of the changes.

**Comment by ipediez** [comment]
Waiting on contributor communications until Wednesday, as I contacted them two weeks ago for another issue and they haven't responded yet. 

If they do not respond, the dataset is ready to go.

**Comment by idazucchi** [comment]
Karoly changed the submission's status to Exported 
I filled out the import form for release 18

**Comment by Wkt8** [comment]
Reopening as this is the final 'double-pipes' issue which is as a result to project.contributors.laboratory not appearing in the UI.

Alexie and I edited the project in the API and it is ready to be exported + import form to be filled. Currently blocked by ontology release.

Tagging @gabsie for tracking.
